Automattic/wp-calypso
# Jetpack Partner Portal: add the feature flag and required modules for a page to create a WP.com site

Closes https://github.com/Automattic/jetpack-avalon/issues/47

## Proposed Changes

This PR adds the feature flags and required modules for a page to create a WP.com site.

## Testing Instructions

- Open the Jetpack Cloud live link
- Append the URL with `/partner-portal/create-site` and verify the below screen is shown.

<img width="1341" alt="Screenshot 2023-08-25 at 10 25 14 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/fbb3d301-0ea8-46b3-b709-8b13041085cb">

- Verify that the `jetpack/pro-dashboard-wpcom-atomic-hosting` feature flag is enabled only in Dev and Horizon environments and not in Staging & Production.

## Pre-merge Checklist
